src/DotLiquid.Tests/Util/StrFTimeTests.cs

```diff
@@ -23,6 +23,7 @@ public class StrFTimeTests
         [TestCase("%m", ExpectedResult = "01")]
         [TestCase("%M", ExpectedResult = "32")]
         [TestCase("%p", ExpectedResult = "PM")]
+		[TestCase("%s", ExpectedResult = "1326033134")]

```

Comment:
Could you rebase from master to avoid the already merged commit?

Reply:
Done